docs/faq/faq.md

```diff
@@ -85,6 +85,93 @@ def wait_on_workflow(*args):
     sensor.execute()
 ```
 
+## How do I monitor the SLA on a workflow?
+
+### Provide a datetime object with UTC specified as the timezone
+
+```python
+from brickflow.context import ctx
+from brickflow_plugins import SLASensor
+from datetime import datetime, timezone
+
+wf = Workflow(...)
+
+@wf.task
+def sla_sensor(*args):
+    api_token_key = ctx.dbutils.secrets.get("scope", "api_token_key")
+    sensor = SLASensor(
+        monitored_task_name="end",
+        env="dev",
+        data_product="product_name",
+        run_date="2024-01-01",
+        sla_sensor_task_names=["sla_sensor"],
+        expected_sla_timestamp=datetime(2024, 1, 1, 10, 0, 0, tzinfo=pytz.utc),
+        dependency_job_name="target_job_name",
+        databricks_host="https://your_workspace_url.cloud.databricks.com",
+        databricks_token=api_token_key,
+        custom_description="message to provide additional context",
+        slack_webhook_url="https://hooks.slack.com/your/webhook/url",
+        email_params={
```

Comment:
Can we make email_params in the class initialization as a pydantic object or a type object to validate the email_params.

Reply:
Hi, @asingamaneni - updated `sla_sensor.py` to have an EmailParams pydantic object and use that to validate the email params passed to the SLASensor in the initialization. Will throw a `ValidationError` if missing required fields